Keas.Mvc/Views/TeamAdmin/BulkLoadPeople.cshtml

```diff
@@ -13,7 +13,7 @@
 
               <p>Note, this may take quite some time for very large departments. You can re-run the loader to import new employees. It will add any new people as needed. It will not remove anyone, even if their appointment has ended.
 </p>
-        <p>What is IAM? <a href="https://iam.ucdavis.edu/" target="_blank">Identity and Access Management</a> that we use to get user information.</p>
+        <p>What is IAM? <a href="https://iam.ucdavis.edu/" target="blank" ref="noopener noreferrer">Identity and Access Management</a> that we use to get user information.</p>
```

Comment:
@laholstege is `blank` valid instead of `_blank`? I've always used the underscore before.  I think you want to keep the underscores.